runtime/lua/vim/_defaults.lua

```diff
@@ -366,16 +366,14 @@ do
 
     -- Add empty lines
     vim.keymap.set('n', '[<Space>', function()
-      local repeated = vim.fn['repeat']({ '' }, vim.v.count1)
-      local linenr = vim.api.nvim_win_get_cursor(0)[1]
-      vim.api.nvim_buf_set_lines(0, linenr - 1, linenr - 1, true, repeated)
-    end, { desc = 'Add empty line above cursor' })
+      vim.go.operatorfunc = "v:lua.require'vim._unimpaired'.space_above"
```

Comment:
@famiu @bfredl Do we have a tracking issue to allow assigning Lua functions directly to options? If so would be good to mention it here so that we can update this in the future.

Reply:
Not a tracking issue for this specifically, but it is one of the items in #25672

Reply:
If it makes sense, I am happy to add a TODO (for myself or just in general) linking to that issue.

Reply:
Yes please do

Reply:
Done